packages/backend-app-api/config.d.ts

```diff
@@ -33,6 +33,37 @@ export interface Config {
        */
       dangerouslyDisableDefaultAuthPolicy?: boolean;
 
+      /** Controls how to store keys for plugin-to-plugin auth */
+      keyStore?:
```

Comment:
Yep thinking there's a bit of a risk of confusion here regarding what this is for. What do you think about something like `backend.auth.pluginKeyStore`?

Reply:
updated